OwenConferenciasApp/screens/UpcomingEvents.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {View, Text, Alert} from 'react-native';
```

Comment:
Here you are importing a component and not even using it. 😨

Reply:
I use this component in the App.js where I have a tab navigation 

![Screenshot_1594070907](https://user-images.githubusercontent.com/42794917/86848347-6ff01b00-c073-11ea-8b6d-1aefb8f5fa53.png)